### PR TITLE
Folder structure for identity-lifecycles

### PR DESCRIPTION
**Pull request by sherenewso2**
- Move files from 'user-mgt' folder and structure to 'identity-lifecycles' folder according to the finalized structure
- rename files for consistency 
- fix link paths

### CODE REVIEW

File: en/docs/guides/identity-lifecycles/manage-roles-overview.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Following are the operations that can be carried out on user roles. 
+
+- [Add User Roles](../add-user-roles)
+
+- [Update User Roles](../update-roles)
```
Comment (DMHP) [inline en/docs/guides/identity-lifecycles/manage-roles-overview.md:5]: We need to remove this page

File: en/docs/guides/identity-lifecycles/manage-user-overview.md [hunk 2]
```diff
@@ -1,7 +1,19 @@
 # Manage User Operations
 
+<<<<<<< HEAD
```
Comment (DMHP) [inline en/docs/guides/identity-lifecycles/manage-user-overview.md:3]: We need to remove this line

File: en/docs/guides/identity-lifecycles/manage-user-overview.md [hunk 3]
```diff
@@ -1,7 +1,19 @@
...
+This section provides instructions on how to perform the following actions on a user in WSO2 Identity Server and manage the user profile. 
+
+- [List/Search Users](../search-user)
+
+- [Delete Users](../delete-users)
+
+- [View/Update User Profiles](../update-profile)
+
+- [Manage User Attributes](../manage-user-attributes)
+=======
```
Comment (DMHP) [inline en/docs/guides/identity-lifecycles/manage-user-overview.md:13]: Need to remove this

File: en/docs/guides/identity-lifecycles/manage-user-overview.md [hunk 4]
```diff
@@ -1,7 +1,19 @@
...
 This section will guide you through the operations that can be carried out on a user in WSO2 Identity Server.
 
 - [List/Search Users](../../identity-lifecycles/search-users)
 
-- [Delete Users](../../identity-lifecycles/delete-users)
\ No newline at end of file
+- [Delete Users](../../identity-lifecycles/delete-users)
+>>>>>>> 8ee2920b80a90dd9211a5d927afdb1c6d081eaa4
```
Comment (DMHP) [inline en/docs/guides/identity-lifecycles/manage-user-overview.md:19]: Need to remove this

File: en/docs/guides/identity-lifecycles/edit-delete-roles.md [hunk 5]
```diff
@@ -73,59 +73,9 @@ Services.](insert-calling-admin-services)
         </soap:Body>
     </soap:Envelope>
     ```
-    
----
-## Update special role names
```
Comment (DMHP) [inline en/docs/guides/identity-lifecycles/edit-delete-roles.md:78]: This section should not be deleted

**Comment by sherenewso2** [comment]
Comments are addressed in PR https://github.com/wso2/docs-is/pull/1838

**Pull request merged**